From ce5b0dc43575de7a08be8de1781b29bb0cb67d94 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:25:31 -0400
Subject: [redhat] kabi: Adding symbol strncpy_from_user (lib/strncpy_from_user.c)

Message-id: <f97898d07a054fed071b42ccb3942be4d840ac38.1410510773.git.poros@redhat.com>
Patchwork-id: 96082
O-Subject: [PATCH RHEL7.1 211/233] kabi: Adding symbol strncpy_from_user (lib/strncpy_from_user.c)
Bugzilla: 1071099
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1071099
    https://bugzilla.redhat.com/show_bug.cgi?id=1071099

    Description:
    ============
    Used to copy a NULL terminated string from userspace.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/strncpy_from_user
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/strncpy_from_user

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/strncpy_from_user b/redhat/kabi/kabi-module/kabi_ppc64/strncpy_from_user
new file mode 100644
index 0000000..e0f4f46
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/strncpy_from_user
@@ -0,0 +1,2 @@
+#1-
+0x24428be5 strncpy_from_user vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/strncpy_from_user b/redhat/kabi/kabi-module/kabi_x86_64/strncpy_from_user
new file mode 100644
index 0000000..4c77eb3
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/strncpy_from_user
@@ -0,0 +1,2 @@
+#1-
+0x6263e02d strncpy_from_user vmlinux EXPORT_SYMBOL
-- 
1.7.1